client/ayon_deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -651,6 +603,139 @@ def inject_ayon_environment(deadlinePlugin):
         raise
 
 
+def _wait_for_in_progress(job, export_url):
```

Comment:
This implementation already will help, but all processes will try to run extract environments if the first process fails.

Can the variables be renamed from `_url` to `_path`?